pages/docs/manual/latest/overview.mdx

```diff
@@ -220,7 +220,7 @@ Integer Addition                | `23 + 1`                             | `23 + 1
 Float Addition                  | `23.0 +. 1.0`                        | `23.0 + 1.0`
 Integer Division/Multiplication | `2 / 23 * 1`                         | `2 / 23 * 1`
 Float Division/Multiplication   | `2.0 /. 23.0 *. 1.0`                 | `2.0 / 23.0 * 1.0`
-Float Exponentiation            | `2.0 ** 3.0`                         | `Math.pow(3, 4)`
+Float Exponentiation            | `2.0 ** 3.0`                         | `Math.pow(3.0, 4.0)`
 String Concatenation            | `"Hello " ++ "World"`                | `"Hello " + "World"`
```

Comment:
This actually needs to be changed into `Math.pow(2.0,3.0)`

Reply:
🤦‍♂️fixed! Thanks.